### PR TITLE
Create MACS.travis.yml

### PR DESCRIPTION
**Pull request by gururajrkatti**

### CODE REVIEW

**Comment by seth-priya** [comment]
@Siddhesh-Ghadi please review

File: travis-ymls/MACS.travis.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  - amd64
+  - ppc64le
+dist: focal
+
+before_install:
+  - gcc --version
+  - sudo apt-get update -qq -y
+  - sudo apt-get install -yq python3 python3-pip python3-setuptools python3-wheel procps zlib1g zlib1g-dev
+  - pip3 install --upgrade pip --no-cache-dir
+  - pip3 install cython --no-cache-dir
```
Comment (Siddhesh-Ghadi) [inline travis-ymls/MACS.travis.yml:24]: Did we try using `nightly` python which contains `cpython`? @gururajrkatti 

https://docs.travis-ci.com/user/languages/python/#nightly-build-support

Comment (gururajrkatti) [reply travis-ymls/MACS.travis.yml:24]: Hi @Siddhesh-Ghadi I tried using nightly python, it is failing (on amd64 also). Below is the error reported on amd64  "ERROR: Failed building wheel for numpy" . Logs at https://travis-ci.com/github/gururajrkatti/MACS/jobs/506020106. Let me know if I am missing something.

Comment (Siddhesh-Ghadi) [reply travis-ymls/MACS.travis.yml:24]: Looks like `numpy` doesn't support latest version of python yet.
```bash
setup.py:66: RuntimeWarning: NumPy 1.20.3 may not yet support Python 3.10.
```
Lets stick to the method you have used. Thanks

**Review by Siddhesh-Ghadi — APPROVED** [review]

**Comment by Siddhesh-Ghadi** [comment]
LGTM @seth-priya

**Pull request merged**